app/smc/sysbuild.cmake

```diff
@@ -5,7 +5,11 @@
 set(VERSION_FILE ${APP_DIR}/../../VERSION)
 set(VERSION_TYPE BUNDLE)
 include(${ZEPHYR_BASE}/cmake/modules/version.cmake)
-string(REGEX REPLACE "[^0-9]+" "" BUNDLE_VERSION_EXTRA_NUM "${BUNDLE_VERSION_EXTRA}")
+if("${BUNDLE_VERSION_EXTRA}" STREQUAL "")
```

Comment:
It would be ideal to de-duplicate this at some point. Not an immediate problem though.